usvm-core/src/main/kotlin/org/usvm/collection/array/ArrayRegion.kt

```diff
@@ -137,7 +146,7 @@ internal class UArrayMemoryRegion<ArrayType, Sort : USort, USizeSort : USort>(
                 fromSrcIdx, fromDstIdx, toDstIdx, USizeExprKeyInfo()
             )
             val newDstCollection = dstCollection.copyRange(srcCollection, adapter, guard)
-            region.updateAllocatedArray(dstConcrete.address, newDstCollection)
+            region.updateAllocatedArray(dstConcrete.address, newDstCollection, srcRef.uctx.defaultOwnership)
```

Comment:
Array `memcpy`, Set `union` and Map `merge` are `write` operations. So, we need to provide an ownership

Reply:
Accordings to usages, this operations used only in `SymbolicCollection.applyTo`, which is performed on composer memory in `ColllectionId.instantiate`